src/Room.ts

```diff
@@ -0,0 +1,22 @@
+import mongoose from 'mongoose';
+
+interface Room {
+    room_id: string;
```

Comment:
mongodb already has a built in `_id` that is used as an index. performing searches on anything that isn't `_id` will result in a table scan.